src/service/repo/anonymousMessageRepo.ts

```diff
@@ -2,6 +2,7 @@ import anonymousApi from '../anonymousConfig';
 import {AnonymousPostNotification} from '../../../types/repo/AnonymousMessageRepo/AnonymousPostNotificationData';
 import {ChannelData} from '../../../types/repo/ChannelData';
 import {GetstreamChannelType} from './types.d';
+import StorageUtils from '../../utils/storage';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `StorageUtils` is a new addition and it seems to be used correctly for fetching timestamps from storage. This change should improve the efficiency of data fetches by only requesting new data since the last fetch.
```diff
+ import StorageUtils from '../../utils/storage';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->